src/Microsoft.DotNet.Build.Tasks/Targets/publishtest.targets

```diff
@@ -83,7 +83,7 @@
       OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
       Retries="$(CopyRetryCount)"
       RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
-      UseHardlinksIfPossible="true">
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)">
```

Comment:
I'm still confused about OS vs OSGroup, but OSGroup is the one that we seem to use now so updated PR using that.